jekyll/_style/style-lists.adoc

```diff
@@ -0,0 +1,11 @@
+= Using lists
+:page-layout: classic-docs
+:icons: font
+:toc: macro
+:toc-title:
+
+* Use ordered lists for steps in a process. All other lists should be unordered.
+
+* Capitalize the first word of each item in a list.
+
+* If the text in the bullet is a full sentence or completes the intro stem, use a full stop at the end.
```

Comment:
@rosieyohannan do you think "period" might be a bit more universal than full stop? I'm not certain myself but I know I didn't use this term until secondary school at least

Reply:
Oh yes you are probably right!